frontend/src/scenes/insights/Histogram/Histogram.tsx

```diff
@@ -67,17 +72,30 @@ export function Histogram({
     // y-axis gridline scale
     const yAxisGrid = config.axisFn.y(y).tickSize(-config.gridlineTickSize).tickFormat('').ticks(y.ticks().length)
 
+    // Update config to new values if dimensions change
+    useEffect(() => {
+        setConfig(getConfig(layout, width, height))
+    }, [width, height])
```

Comment:
Is the `layout` missing from the deps array intentional? Seems to be reading the comment above the effect, but it would be good to explicitly mark it out, as the default suggestion is to always have all variables that you use in the array.

Reply:
Good catch! I think I initially left it out because layout state wasn't being tracked in kea state yet, but it doesn't make sense not to include it here.